common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -35,4 +33,4 @@
   % if msg:
       <span class="message">${HTML(msg)}</span>
   % endif
-</div>
+</form>
```

Comment:
@muzaffaryousaf no, this was the opposite of what I was trying to suggest.  We only want one form element for each Problem, instead of for each responsetype.  Do you think we can do that?  So, if you author a multi-responsetype problem, there should only be one form, and one "Check" button.